### PR TITLE
remove all non-working code

### PR DESCRIPTION
**Pull request by gregglind**

### CODE REVIEW

File: addon/Config.jsm [hunk 1]
```diff
@@ -72,9 +72,6 @@ var config = {
   // required LOG key
   "log": {
     // Fatal: 70, Error: 60, Warn: 50, Info: 40, Config: 30, Debug: 20, Trace: 10, All: -1,
-    "bootstrap":  {
-      "level": "Debug",
```
Comment (biancadanforth) [inline addon/Config.jsm]: Required for what? I know this goes to Log.jsm in StudyUtils... Not clear to me though why this is here?

File: addon/bootstrap.js [hunk 2]
```diff
@@ -17,12 +18,16 @@ const { studyUtils } = Cu.import(STUDYUTILSPATH, {});
 
 const REASONS = studyUtils.REASONS;
 
-// QA NOTE: Study Specific Modules - package.json:addon.chromeResouce
-const BASE = `template-shield-study-button-study`;
-XPCOMUtils.defineLazyModuleGetter(this, "Feature", `resource://${BASE}/lib/Feature.jsm`);
+// logging
```
Comment (biancadanforth) [inline addon/bootstrap.js]: This seems very similar/identical to studyUtils.createLog? Why not call it instead? I'm still not sure what the log is for -- perhaps explain as a comment?

File: addon/bootstrap.js [hunk 3]
```diff
@@ -42,7 +47,7 @@ XPCOMUtils.defineLazyModuleGetter(this, "Feature", `resource://${BASE}/lib/Featu
 
 async function startup(addonData, reason) {
   // `addonData`: Array [ "id", "version", "installPath", "resourceURI", "instanceID", "webExtension" ]  bootstrap.js:48
-  console.log("startup", REASONS[reason] || reason);
+  log.info("startup", REASONS[reason] || reason);
```
Comment (biancadanforth) [inline addon/bootstrap.js]: How is `log.info` better than/different from `console.log`?

File: addon/bootstrap.js [hunk 4]
```diff
@@ -109,13 +114,13 @@ async function startup(addonData, reason) {
   * studyUtils._isEnding means this is a '2nd shutdown'.
   */
 function shutdown(addonData, reason) {
-  console.log("shutdown", REASONS[reason] || reason);
+  log.info("shutdown", REASONS[reason] || reason);
```
Comment (biancadanforth) [inline addon/bootstrap.js]: Why do we need to distinguish all these types of shutdowns? Perhaps give an example for each `if` statement of what you might do differently in that case?

File: addon/lib/Feature.jsm [hunk 5]
```diff
@@ -97,24 +97,28 @@ class Feature {
...
           });
-          feature.studyUtils.endStudy("");
+          feature.studyUtils.endStudy("introduction-leave-study");
         },
       }],
       // callback for nb events
       null
     );
+
+    notice.setAttribute("data-study-config", JSON.stringify(this.variation));
```
Comment (biancadanforth) [inline addon/lib/Feature.jsm]: This looks to be exclusively for testing -- perhaps note that?

File: addon/Config.jsm (hunk 1 above)
Comment (gregglind) [reply addon/Config.jsm]: (I already pulled this out, and switched over to proper logging.

File: addon/bootstrap.js (hunk 2 above)
Comment (gregglind) [reply addon/bootstrap.js]: Hm. you are identifiying much that is mysterious.  I will try setting up CI soon.

File: addon/bootstrap.js (hunk 3 above)
Comment (gregglind) [reply addon/bootstrap.js]: I changed these to log.debug, which is kind of the point.  It's pref controllable now, so the bootstrap logs can actually be silenced.

File: addon/bootstrap.js (hunk 4 above)
Comment (gregglind) [reply addon/bootstrap.js]: I welcome specific language.

File: addon/lib/Feature.jsm (hunk 5 above)
Comment (gregglind) [reply addon/lib/Feature.jsm]: Done.

File: addon/bootstrap.js (hunk 4 above)
Comment (biancadanforth) [reply addon/bootstrap.js]: Well that's the thing, as a new shield developer, I'm not sure myself what kinds of things I might want/need to do for each special case.

**Pull request merged**